experiment/resultstore/main.go

```diff
@@ -286,30 +395,195 @@ func setup(account, tok, invID string) error {
 			},
 		},
 	})
-
-	print("cct", ct, err)
+	if err != nil {
+		return fmt.Errorf("create configured target: %v", err)
+	}
+	print("cct", ct)
```

Comment:
Can you clarify some of the 1-3 letter variables/log lines in this file? Some of them are quite far from their definition, or hard to read/remember :(

Reply:
I promise I'll clean this up and use better variable names (often with multiple syllables!) once I'm refactoring it for others to use.